tests/unit/expire.tcl

```diff
@@ -601,4 +601,117 @@ start_server {tags {"expire"}} {
            {del foo}
         }
     } {} {needs:repl}
+
+    test {EXPIRE with NX option on a key with ttl} {
+        r SET foo bar EX 100000
+        assert_equal [r EXPIRE foo 200000 NX] 0
+        assert_range [r TTL foo] 50000 100000
```

Comment:
i think it'll be slightly more readable if we convert the constants that are in seconds to values in the range of `100`, and only use 100,000 for the ones that are in milliseconds.
```suggestion
        r SET foo bar EX 100
        assert_equal [r EXPIRE foo 200 NX] 0
        assert_range [r TTL foo] 50 100
```
same for the rest of the tests below.